### PR TITLE
 Adds Swiss Federal Geoportal base layers

### PR DESCRIPTION
**Pull request by procrastinatio**
The following layers are to be added:

swiss_geoportal_15

The layer `ctx.providers.SwissFederalGeoportal.JourneyThroughTime` has an optional parameter `time` which controls the displayed year. Currently, all years between 1844 and 2018 are available (as `18441231`). Note, 1964 is the first year the map covers Switzerland entirely.
```
 layer = ctx.providers.SwissFederalGeoportal.JourneyThroughTime
 layer['time'] = '20181231'    # i.e. December, 31st 2018
```
zeitreise-10

Addressing issue https://github.com/geopandas/xyzservices/issues/99

### CODE REVIEW

File: xyzservices/data/providers.json [hunk 1]
```diff
@@ -2989,5 +2989,80 @@
...
+            "html_attribution": "<a target=\"_blank\" href=\"https://www.swisstopo.admin.ch/\">swisstopo</a>",
+            "attribution": "© swisstopo",
+            "bounds": [
+                [
+                    5.140242,
+                    45.398181
+                ],
+                [
+                    11.47757,
+                    48.230651
```
Comment (martinfleis) [inline xyzservices/data/providers.json:3005]: The order of bounds is opposite. It should be lat, lon. You can check the other Swiss dataset on the top of the file.

**Comment by martinfleis** [comment]
Thanks a lot! I'll make a release during the day, to have it available.

**Comment by davidoesch** [comment]
Thx ! Excellent—- as soon as it’s available we will spread the news

**Comment by martinfleis** [comment]
It is already available if you update `xyzservices` to the version `2022.1.0`. Both pip and conda now have it.

```py
In [1]: import xyzservices.providers as xyz

In [2]: xyz.SwissFederalGeoportal
```

**Comment by drnextgis** [comment]
Swiss Federal Geoportal base layers now in QMS catalog:

1. [SWISSIMAGE](https://qms.nextgis.com/geoservices/4683/)
2. [NationalMapGrey](https://qms.nextgis.com/geoservices/4682/)
3. [NationalMapColor](https://qms.nextgis.com/geoservices/4681/)
4. [JourneyThroughTime](https://qms.nextgis.com/geoservices/4684/)

image

**Pull request merged**